openedx/features/partners/views.py

```diff
@@ -38,3 +44,24 @@ def register_user(request, slug):
         return views.Give2AsiaRegistrationView.as_view()(request, partner=partner)
     except ImportError:
         raise Http404('Your partner is not properly registered')
+
+
+@require_http_methods(['POST'])
+def reset_password_view(request):
+    """
+    This is the basic password reset view, as per the requirements
+    of organization password reset flow. Have to send 404 if user does
+    not exist
+    :param request: The HttpRequest request object.
+    :return: HTTPResponse/JSONResponse object with success/error status code
+    """
+    email = request.POST.get('email')
+    reset_password_form = PartnerResetPasswordForm(data={'email': email})
+    if reset_password_form.is_valid():
+            response = password_change_request_handler(request)
```

Comment:
It seems this code block is not properly indented

Reply:
Fixed. Thanks for pointing it out.